NEWS.md

```diff
@@ -4,18 +4,20 @@ This release adds a number of helper functions, primarily for use with NONMEM mo
 
 ## New features and changes
 
+* Added `nm_file()`, `nm_grd()`, `nm_tab()`, `nm_par_tab()`, and `nm_data()` for reading in NONMEM files more easily. (#426)
+
+* Added `nm_join()`, `nm_tables()`, and `nm_table_files()` for reading in NONMEM tables more easily. Notably, `nm_join()` can be used to get a single tibble containing your NONMEM input data joined against all of your table outputs. (#429 and #430)
+
 * `param_estimates_batch()` for extracting a tibble of parameter estimates from a batch of NONMEM runs. Especially useful for large batches of runs created by something like a bootstrap. (#386)
 
 * `cov_cor()` and `check_cor_threshold()` for pulling in covariance and correlation matrices from NONMEM `.cov` and `.cor` files. (#414)
 
+* `copy_model_from(.new_model)` is now an optional argument. By default, it now tries to increment to the next available integer in the destination directory. (#424)
```

Comment:
I had to reread the first sentence to understand it.  It'd might just be me, but I'd find either of these clearer: 

* The `.new_model` argument of `copy_model_from()` is now optional
* Passing the `.new_model` argument to `copy_model_from()` is now optional